### PR TITLE
fixes #23747 - correct filter tags

### PR DESCRIPTION
**Pull request by thomasmckay**
To test:
+ sync busybox
+ create content view with filter only "latest" tag
+ publish
+ confirm just one tag, "latest"

### CODE REVIEW

**Comment by sjha4** [comment]
With my limited knowledge around Container Image filters, this is what I set up:
taglatest

Successfully tested with BusyBox repo with filter 1.26* resulting in 25 Container Image Manifests and 16 Container Image Tags.

After publish, my cv has only the filtered tags:
cv_tag_versions

Code looks as per behavior if my test setup is correct. Initially (mis)understood that a filter with the literal word "latest" is supported.. :smiley: 

Works as expected if my test setup looks correct to you.

**Comment by thomasmckay** [comment]
@sjha4 - interesting! Let me try it out. Maybe _ILIKE_ is incorrect.

**Comment by thomasmckay** [comment]
view docker tags for repository busybox

I added two filters: 1.25, 1.26.1*

I had forgotten about the wildcard patterns; glad to see they work!

**Comment by sjha4** [comment]
Unless you are making any more functional changes, looks good to me..ACK:+1: (without commit access)

File: app/models/katello/content_view_docker_filter.rb [hunk 1]
```diff
@@ -32,7 +32,7 @@ def fetch_tag_uuids(repo, rule, manifest_klass = DockerManifest)
 
       query = manifest_klass.in_repositories(repo).where("id in (#{tags_query.to_sql})")
       names = query.all.collect do |manifest|
-        manifest.docker_tags.where(:repository => repo).all.collect do |tag|
+        manifest.docker_tags.where(:repository => repo).where("name ilike ?", query_name).all.collect do |tag|
```
Comment (parthaa) [inline app/models/katello/content_view_docker_filter.rb]: can you add a line that does the following
```
query_name = query_name.name.tr("*", "%")
```

Comment (thomasmckay) [reply app/models/katello/content_view_docker_filter.rb]: @parthaa - I think that is already there?
https://github.com/Katello/katello/pull/7464/files#diff-c5101bb5147bb92312d8f0a90c94ead5R27

Comment (parthaa) [reply app/models/katello/content_view_docker_filter.rb]: good point.

**Comment by parthaa** [comment]
ack

**Pull request merged**